### PR TITLE
The gateway is now indestructible

### PR DESCRIPTION
**Pull request by ChangelingRain**
:cl: Joan
rscadd: The gateway is now indestructible, short of admin shenanigans.
tweak: The bottom three parts of the gateway are all nondense.
/:cl:

To make it destructible, varedit the `destructible` var to 1
It's also now immune to singularity pulling.

### CODE REVIEW

**Comment by KorPhaeron** [comment]
But why?

**Comment by xxalpha** [comment]
I disagree with this, I think people should be stranded if something destroys the gateway.

**Comment by KazeEspada** [comment]
Only one thing can destroy the gateway.

**Comment by Shadowlight213** [comment]
:-1: If you're worried about being stranded, stay on the station.

File: code/modules/awaymissions/gateway.dm [hunk 1]
```diff
@@ -2,13 +2,23 @@ var/obj/machinery/gateway/centerstation/the_gateway = null
 
 /obj/machinery/gateway
 	name = "gateway"
-	desc = "A mysterious gateway built by unknown hands, it allows for faster than light travel to far-flung locations."
+	desc = "A mysterious, seemingly indestructible, gateway built by unknown hands, it allows for faster than light travel to far-flung locations."
```
Comment (RandomMarine) [inline code/modules/awaymissions/gateway.dm]: That comma after indestructible should be axed.

**Comment by Incoming5643** [comment]
Ironically this makes the center of a deactivated gateway a really appetizing blob spawn as 8 out of the 9 surrounding tiles are utterly invulnerably to attack

**Comment by ChangelingRain** [comment]
Can't spawn in the center, as it's dense.

**Comment by PKPenguin321** [comment]
:-1:

**Comment by MMMiracles** [comment]
I remember this one time I was with a party of people on the cave's mission, I was ahead of like 7-10 people and at the gateway end. I threw a grenade to take out the skeleton spawner and the blast broke the mid-section of the gateway.

I managed to accidentally leave like 10 people stranded in hell after I shot myself in shame.

This change removes any hope of people in the future feeling the same shame as me and that is a bad thing. :-1:

**Comment by Niknakflak** [comment]
:-1: yea tbh

**Pull request closed without merging**